images/metering-ansible-operator/roles/meteringconfig/tasks/validate.yml

```diff
@@ -87,21 +87,3 @@
       status: "True"
       message: "Finished the validation process"
       lastTransitionTime: "{{ now(utc=False).isoformat() + 'Z' }}"
-
-- name: Log Events for validating configurations
```

Comment:
How come this section goes away without a replacement event?

Reply:
I didn't see much value in firing off an event in the current state of this task file, but I'll re-add it now.